lib/rollbar/request_data_extractor.rb

```diff
@@ -28,7 +28,12 @@ def extract_request_data_from_rack(env)
       session = rollbar_filtered_params(sensitive_params, rollbar_request_session(rack_req))
       route_params = rollbar_filtered_params(sensitive_params, rollbar_route_params(env))
 
-      params = request_params.merge(get_params).merge(post_params).merge(raw_body_params)
+      params =
+        if !raw_body_params.empty?
```

Comment:
@elight this change is not related to the PR title. We may probably discuss this in another one.